packages/mask/src/plugins/Trader/SNSAdaptor/trader/TraderDialog.tsx

```diff
@@ -118,7 +118,7 @@ export function TraderDialog({ open, onClose }: TraderDialogProps) {
                                 chains={chainIdList}
                             />
                         </div>
-                        <Trader {...traderProps} chainId={chainId} classes={{ root: classes.tradeRoot }} />
+                        <Trader chainId={chainId} {...traderProps} classes={{ root: classes.tradeRoot }} />
```

Comment:
This will change the original behavior. Why are you need this?

Reply:
We need this to force the user to trade(swap) only on the Ethereum network.
Oracle does not detect transactions on other networks, and the user will receive a reward only if he purchases a token on the Ethereum network.

Expected behaviour: 


https://user-images.githubusercontent.com/32291396/165485248-a18ce5a4-a067-4119-9454-94b4bf4c41a2.mov

I also thought it would change the initial behavior, but surprisingly, `traderProps` has `chainId`, but `traderProps.chainId` is always overwritten by the current `chainId`.

Reply:
Got it.